docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -1161,6 +1161,29 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
             Scripts</link>) for more details.</para>
           </listitem>
         </orderedlist>
+
+        <sect3 id="Addl_SSH_KeyInfo">
+          <title>Additional SSH Key Information</title>
+
+          <para>On a multi-node HPCC system, it is quite important that the
+          certificates and SSH keys all match across all nodes in order for
+          the system to work properly. If you used the
```

Comment:
it is quite important that the certificates and SSH keys all match across all nodes in order for the system to work properly.
s/b
certificates and SSH keys **must** match across all nodes in order for the system to work properly.